aries-site/src/pages/index.js

```diff
@@ -13,8 +25,41 @@ const Index = () => {
     }
   }, []);
 
-  // return <Landing />;
-  return <About />;
+  return (
+    <PageLayout title="Home" isLanding>
+      {/* <Grid columns={['flex', 'xxlarge', 'flex']}> */}
+      {/* <Box /> */}
```

Comment:
I'm keeping these comments since I might want to explore it more after we get the Nav settled.